mixer/pkg/config/crd/store.go

```diff
@@ -32,13 +32,6 @@ import (
 	"istio.io/istio/pkg/probe"
 )
 
-const (
-	// API group / version for istio config.
-	apiGroup        = "config.istio.io"
-	apiVersion      = "v1alpha2"
-	apiGroupVersion = apiGroup + "/" + apiVersion
-)
-
```

Comment:
@douglas-reid / @geeknoid et al was there any reason to have this in a generic config.istio.io instead of <subsystem>.istio.io ?